content/hi/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+title: ऐप कंटेनर (App Container)
+id: app-container
+date: 2019-02-12
+full_link:
+short_description: >
+  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
```

Comment:
I feel  कार्यभार is a literal machine translation & we shouldn't be using it.

Reply:
I take reference from Korean and cncf glossary localization. They also used as same like this.

Reply:
What do you think @divya-mohan0209

Reply:
@taniaduggal I would suggest:
```suggestion
  कार्यभार का एक हिस्सा चलाने के लिए प्रयोग किया जाने वाला कंटेनर। इनिट (init) कंटेनर के साथ तुलना करें।
```